setup.py

```diff
@@ -42,6 +42,8 @@ def find_version(*file_paths):
         "Jinja2>=2.10",
         "jsonschema>=3.0.1",
         "pytest>=4.5.0",
+        "pytest-random-order>=1.0.4",
+        "pytest-localserver>=0.5.0",
```

Comment:
How is this related to pre-commit?

Reply:
`pytest` runs in pre-commit and wasn't passing because of missing dependencies - something in the chain must have changed.